portal/plugins/gnmatomresponder/master_importer.py

```diff
@@ -179,6 +179,23 @@ def import_new_item(self, master_item, content, parent=None):
         from pac_xml import PacXmlProcessor
         from mock import MagicMock
         if not isinstance(master_item, VSItem) and not isinstance(master_item, MagicMock): raise TypeError #for intellij
+        from portal.plugins.kinesisresponder.sentry import inform_sentry_exception
+
+        vs_item_id = master_item.get("itemId")
+
+        try:
+            importjob = ImportJob.objects.get(item_id=vs_item_id)
+        except:
```

Comment:
make this `except ImportJob.DoesNotExist` instead, if there is some other exception here we don't want it to get ignored

Reply:
Done.